docs/guide/localization/static.md

```diff
@@ -261,19 +335,19 @@ Here is an example module `index.js` file that sets the `browser: true` flag for
 
 <AposCodeBlock>
 
-  ```javascript
-  module.exports = {
-    // Note this is at top level, not under options
-    i18n: {
-      ourTeamUI: {
-        browser: true
-      }
+```javascript
+module.exports = {
+  // Note this is at top level, not under options
+  i18n: {
+    ourTeamUI: {
+      browser: true
     }
-  };
-  ```
-  <template v-slot:caption>
-    /modules/team-ui/index.js
-  </template>
+  }
+};
+```
+<template v-slot:caption>
+/modules/team-ui/index.js
```

Comment:
I think you should harmonize this with your venue strings example earlier.

Reply:
Fixed/updated.